src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -37,6 +37,7 @@
         display: flex;
         flex: 1 1 auto;
         flex-direction: column;
+        width: 100%;
```

Comment:
Maybe just add a comment for posterity -- something about how since it's flex-direction column, we want to explicitly set the width